.github/workflows/release.yml

```diff
@@ -69,7 +68,7 @@ jobs:
         run: gzip -c "./extra/alacritty.man" > "./alacritty.1.gz"
       - name: Upload Assets
         run: |
-          cp ./extra/logo/alacritty-term.svg ./Alacritty.svg
+          mv ./extra/logo/alacritty-term.svg ./Alacritty.svg
```

Comment:
I mean, why should we even move it before upload? Can't we use full path like you do for everything else?

Reply:
To rename it?

Reply:
Oh, yeah, forgot that it's different name..